linebot/webhook/parse.go

```diff
@@ -33,6 +38,10 @@ func ParseRequest(channelSecret string, r *http.Request) (*CallbackRequest, erro
 	return &cb, nil
 }
 
+func ParseRequest(channelSecret string, r *http.Request) (*CallbackRequest, error) {
+	return ParseRequestWithOption(channelSecret, r, nil)
+}
+
```

Comment:
Can you do these like the Java SDK?

- Write comments explaining why users need this option in `ParseRequestWithOption` /when it should be used.
- Indicate to `ParseRequest` users that there is a version where the option can be utilized.
- Write sufficient tests.